package.json

```diff
@@ -52,8 +52,8 @@
     "bundle": "lerna run bundle",
     "watch": "lerna run --parallel watch",
     "test": "lerna run test",
-    "electron": "yarn --cwd applications/electron",
-    "browser": "yarn --cwd applications/browser",
```

Comment:
Those should not be adapted. The idea is to call `yarn electron start` in the root directory

Reply:
Thanks for the comment. I'll upload a new version of the PR.